site/app/templates/admin/UploadConfigForm.twig

```diff
@@ -26,6 +26,8 @@
         Upload Config: <input type="file" name="config_upload" /><br />
         <input type="submit" value="Upload" />
     </form>
+    <br>
+    <a class="btn btn-primary" style="margin-top: 6px" href={{ back_url }}>Go Back To AutoGrading Page</a>
```

Comment:
Capitalization should be `Autograding` (lower case g)

Reply:
Let's also hide the button if `{{ back_url }}` is blank (with change below) as it does not make sense.